src/test/kotlin/fr/social/gouv/agora/usecase/reponseConsultation/InsertReponseConsultationUseCaseTest.kt

```diff
@@ -79,8 +79,10 @@ internal class InsertReponseConsultationUseCaseTest {
     @Test
     fun `insertReponseConsultation - when consultation is already finished - should return failure`() {
         // Given
-        val consultationFinished = consultationInfo.copy(endDate = LocalDateTime.now().minusDays(30).toDate())
-
+        val todayDate = LocalDateTime.of(2023, Month.OCTOBER, 19, 19, 0, 0)
+        val endDate = LocalDateTime.of(2023, Month.JUNE, 20, 14, 0, 0)
+        val consultationFinished = consultationInfo.copy(endDate = endDate.toDate())
+        mockDate(todayDate)
```

Comment:
Tu peux faire directement `mockDate(LocalDateTime.of(...))`

Reply:
done